src/stores/pebbleLupa.js

```diff
@@ -0,0 +1,21 @@
+import { createStore, createHook } from "react-sweet-state";
+import { execute } from "./utils/loadFromBackend";
+
+const Store = createStore({
+  initialState: {},
+  actions: {
+    load: ({uuid}) => ({ getState, setState }) => {
+      return execute(
+        { getState, setState },
+        {
+          key: "pebbleLupa",
+          urlEnding: `${uuid}`
```

Comment:
Why is uuid in `${ }` ? Why not just urlEnding: uuid ?

Reply:
Was copy and replace from lupa equivalent with ytunnus in the same place. Didn't think through. Anyway the file wasn't supposed to be added and was removed already.